openslides/agenda/static/js/agenda/site.js

```diff
@@ -268,13 +269,19 @@ angular.module('OpenSlidesApp.agenda.site', [
                     return false;
             }
         };
-        $scope.getUpdateStatePrefix = function (item) {
+        $scope.getDetailStatePrefix = function (item) {
             var prefix = item.content_object.collection.replace('/','.');
             // Hotfix for Issue 2566.
             // The changes could be reverted if Issue 2480 is closed.
```

Comment:
Yes, it is. The collection from a motion block is `motions/motion-block`. For the states and forms we need `motions/motionBlock`.